### PR TITLE
Only export root filesystems to one IP per VM

### PR DESCRIPTION
**Pull request by SlothOfAnarchy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,11 +51,11 @@ This table only lists the options used in this role, see [xen-vman](https://gith
...
+| `name`           |    :heavy_check_mark:    | `name`-suffix of the filesystem. The final name consists of the vm name and the filesystem name, delimited by a dash.                                                                                                                                                                                                                          |
+| `zfs_attributes` |           `{}`           | A dict containing any ZFS attributes that shall be set for this filesystem. It is recommended to define a `quota`, though this may also be done in the default ZFS attributes on the actual [zfs-storage](https://github.com/stuvusIT/zfs-storage/) server.                                                                                    |
+| `nfs_options`    | `[rw=@{{ansible_host}}]` | A list of NFS options that are set for this NFS export. The default is one read-write NFS export for the IP defined in (`ansible_host`) in the hostvars of the respective VM. The options have to conform to the ZFS `sharenfs` attribute format. The options defined in `vm_nfs_options` will be set in addition to this value in every case. |
```
Comment (Mr-Pi) [inline README.md:58]: why the fqdn and not ansible_host_short?

File: scripts/storage-facts.py [hunk 2]
```diff
@@ -110,11 +110,9 @@ def generateFacts(original_facts, storage_host):
...
       # Default NFS options if no overrides are specified for a filesystem
       default_nfs_options = list(nfs_options)
-      # Add every existing interface ip as rw export
-      if 'interfaces' in original_facts[host]:
-        for interface in original_facts[host]['interfaces']:
-          if 'ip' in interface:
-            default_nfs_options.append("rw=@" + interface['ip'])
+      # Add VM IP as rw export
+      if 'ansible_host' in original_facts[host]:
+        default_nfs_options.append("rw=@" + original_facts[host]['ansible_host'])
```
Comment (Mr-Pi) [inline scripts/storage-facts.py:115]: see above

**Review by Mr-Pi — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (SlothOfAnarchy) [reply README.md:58]: `ansible_host` is the IP of the respective host.

Comment (dasJ) [reply README.md:58]: @Mr-Pi From the inventory:

```
hostConfig['ansible_host'] = interface['ip'].split('/')[0]
```

**Review by Mr-Pi — APPROVED** [review]

**Pull request merged**